dj_hetmech_app/serializers.py

```diff
@@ -13,18 +13,8 @@ class Meta:
         exclude = ('id', 'metapath', )
 
 
-class CompactMetaPathSerilizer(serializers.ModelSerializer):
-    """This serializer only includes two fields in Metapath model.
-    It is defined specifically for PathCountDgpSerializer.
-    """
-    class Meta:
-        model = Metapath
-        fields = ('abbreviation', 'name', )
-
-
 class PathCountDgpSerializer(serializers.ModelSerializer):
     dgp = DgpSerializer()
-    metapath = CompactMetaPathSerilizer()
```

Comment:
You probably also have to update the polisher to have the key rather than the dict as the metapath field?

Reply:
Already did.